src/_includes/header.html

```diff
@@ -1,5 +1,5 @@
-<header class="site-header">
-  <nav class="navbar navbar-expand-lg navbar-light justify-content-between">
+<header class="flutter-header">
```

Comment:
Could we stick with `site-foo` instead of `flutter-foo`. For main site classes like this, it would encourage reuse across sites. (While the site theme might change across sites, I'd expect, for example, `site-header` to be used everywhere.)